vnext/.editorconfig

```diff
@@ -14,7 +14,7 @@ indent_size = 2
 end_of_line = crlf
 
 # Xml project files
-[*.{csproj,vcxproj,pssproj,vcxproj.filters,targets,props}]
+[*.{csproj,vcxproj,pssproj,vcxproj.filters,vcxitems,targets,props}]
```

Comment:
>vcxitems [](start = 43, length = 8)

(nit) can we alphabenitze those?

Reply:
Sorry! Missed this comment.
Will sort these in a subsequent PR.